test/mocha/30-nonce.js

```diff
@@ -124,5 +124,49 @@ describe('Nonce API', () => {
       err.message.should.equal('Invalid "entryStyle" "not-machine-or-human"; ' +
         '"entryStyle" must be "human" or "machine".');
     });
+    it('should generate a new nonce if one already exists', async () => {
+      const accountId = mockData.accounts['alpha@example.com'].account.id;
+      const actor = await brAccount.getCapabilities({id: accountId});
+      let err;
+      let result1;
+      let result2;
```

Comment:
as a best practice, you want to define variables close to where they are used.  So this should be moved down to around L157.